src/components/HomePage/Announcement.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Card, CardActionArea, CardContent, Grid, Typography } from '@material-ui/core';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForwardIos';
+
+import LinkNoStyle from '../LinkNoStyle/LinkNoStyle';
+import { graphql, useStaticQuery } from 'gatsby';
+
+const styles = theme => ({
+	container: {
+		backgroundColor: '#FFFFFF',
+		padding: theme.spacing(2),
+		justifyContent: 'center'
+	},
+	card: {
+		width: '100%',
+		borderRadius: '20px',
+		boxShadow: '0 1px 20px rgba(201, 96, 255, 0.5)',
+		background: 'linear-gradient(90deg, #FFFFFF, #F1D6FF, #FFFFFF, #F1D6FF)',
+		position: 'relative',
+		animation: `$swipe 3s linear infinite`
+	},
```

Comment:
Can we play around with the colors a little? Maybe our accent color (pink or something?) because to me the purple blends in a little too much with the banner. Maybe if the pink doesn't look good, then we can try adding more padding to add visual separation between the banner and announcement?